sqlstore/src/it/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateServiceRecordDaoIT.java

```diff
@@ -73,45 +74,47 @@ public void setup() throws IOException {
     emptySR.setId(2L);
     Mockito.when(instrumentDao.get(ArgumentMatchers.anyLong())).thenReturn(emptySR);
   }
-  
+
   @Test
   public void testSaveNew() throws IOException {
+
     String title = "New Record 1";
     Long newId = dao.save(makeServiceRecord(title));
-    
+
     ServiceRecord savedRec = dao.get(newId);
     assertEquals(title, savedRec.getTitle());
   }
-  
+
   private ServiceRecord makeServiceRecord(String title) {
     ServiceRecord rec = new ServiceRecord();
     rec.setTitle(title);
-    rec.setInstrument(emptySR);
+    instrumentService.addServiceRecord(rec, emptySR);
```

Comment:
This method is meant to create a ServiceRecord Java object. It should not be saving in the DB. Since instrument is not a ServiceRecord field anymore, the instrument should not be set here.